templates/docs/examples/layouts/_global-nav.html

```diff
@@ -1,7 +1,3 @@
-{# this is a mockup of global nav for testing, based on global-nav@2.4.1 #}
```

Comment:
Would be good to leave this comment above the HTML part of global navigation, because it's generated as well and shouldn't be manually added.
Would be nice to somehow get rid of the HTML as well, but may be tricky to do (other than using real global nav injected into the page).

Reply:
Cool - I've gone for the comment.